lib/badge_queries.rb

```diff
@@ -99,6 +99,15 @@ module BadgeQueries
   GROUP BY p.user_id
 SQL
 
+  WikiEditor = <<SQL
```

Comment:
The [squiggly heredoc](https://infinum.co/the-capsized-eight/multiline-strings-ruby-2-3-0-the-squiggly-heredoc) is better for multiline strings ;)

```suggestion
  WikiEditor = <<~SQL
```